test/cppsuite/tests/cursor_bound_01.cpp

```diff
@@ -504,6 +502,9 @@ class cursor_bound_01 : public test {
         } else
             ret = normal_cursor->next(normal_cursor.get());
 
+        if (ret == WT_ROLLBACK)
```

Comment:
For consistency with the rest of the code, can we assert `ret` can be equal to `WT_ROLLBACK` and call `return` if that's the case?
```suggestion
        testutil_assert(ret == 0 || ret == WT_NOTFOUND || ret == WT_ROLLBACK);
        if (ret == WT_ROLLBACK)
          return;
```

Reply:
e46764d